JakinChan200/CS180_Roald
# #27 Video Publish Time Form

## Overview
Fixes the KR page and adds it again to the navbar
Adds 'Experimental Metrics' and 'Submit Video Data' dropdown sections
Adds animations to dropdown contents
Adds scrolling to video results
Adds hover tooltips to use on form for more field descriptions
Adds form with username/password optional fields that show save/delete
Adds publish time field that is shown regardless in 'Experimental Metrics'

To do:
Once data is actually pulled from the endpoints, change the setStates on expResults to push that result to the existing list instead of replacing

## Materials
- Issue #27 


## Pictures
![fullcountrydemo](https://user-images.githubusercontent.com/43893085/163888064-32adef68-cf75-495e-962d-edfdbc7f7347.gif)
![image](https://user-images.githubusercontent.com/43893085/163888105-80e70344-a590-4bfb-ab2c-916bbd027c96.png)
![image](https://user-images.githubusercontent.com/43893085/163888113-8be99b83-2a64-499b-a64a-767a9a453500.png)
![image](https://user-images.githubusercontent.com/43893085/163888129-eb44e8e6-5886-4495-82c4-a3a9ba1d1c03.png)

## Manual Testing Guide
- Navigate to the country pages
- Note that some of the collapsible bars now are closed by default
- Open the 'Submit Video Data' section and note the form exists
- Note that hovering over the hourglass shows tooltips on what the field does
- Note that changing the publish_time changes the results in 'Experimental Metrics'
- Note that clicking the save or delete buttons gives an error currently
